### ISSUE TITLE
Error when accessing empty array in `/courses/abbr` route

### ISSUE THREAD

**Issue by rutgerfarry** [classmere/api#9]
When the array of results from Postgres returns empty, the app crashes

```
08:31:33 web.1  | /Users/rutger/Dropbox/projects/classmere/api/routes/courses.js:45
08:31:33 web.1  |         title: data[0].title,
08:31:33 web.1  |                       ^
08:31:33 web.1  | TypeError: Cannot read property 'title' of undefined
08:31:33 web.1  |     at null.<anonymous> (/Users/rutger/Dropbox/projects/classmere/api/routes/courses.js:45:23)
08:31:33 web.1  |     at emit (events.js:107:17)
08:31:33 web.1  |     at after (/Users/rutger/Dropbox/projects/classmere/api/node_modules/pg/lib/native/query.js:76:10)
08:31:33 web.1  |     at onResult (/Users/rutger/Dropbox/projects/classmere/api/node_modules/pg-native/index.js:166:5)
08:31:33 web.1  |     at g (events.js:199:16)
08:31:33 web.1  |     at emit (events.js:107:17)
08:31:33 web.1  |     at Client._read (/Users/rutger/Dropbox/projects/classmere/api/node_modules/pg-native/index.js:126:12)
08:31:33 web.1  |     at PQ.emit (events.js:104:17)
08:31:33 web.1  |
08:31:33 web.1  | npm ERR! Darwin 14.4.0
08:31:33 web.1  | npm ERR! argv "node" "/usr/local/bin/npm" "start"
08:31:33 web.1  | npm ERR! node v0.12.7
08:31:33 web.1  | npm ERR! npm  v2.12.1
08:31:33 web.1  | npm ERR! code ELIFECYCLE
08:31:33 web.1  | npm ERR! web@0.0.0 start: `node --harmony ./bin/www`
08:31:33 web.1  | npm ERR! Exit status 1
08:31:33 web.1  | npm ERR!
08:31:33 web.1  | npm ERR! Failed at the web@0.0.0 start script 'node --harmony ./bin/www'.
08:31:33 web.1  | npm ERR! This is most likely a problem with the web package,
08:31:33 web.1  | npm ERR! not with npm itself.
08:31:33 web.1  | npm ERR! Tell the author that this fails on your system:
08:31:33 web.1  | npm ERR!     node --harmony ./bin/www
08:31:33 web.1  | npm ERR! You can get their info via:
08:31:33 web.1  | npm ERR!     npm owner ls web
08:31:33 web.1  | npm ERR! There is likely additional logging output above.
08:31:33 web.1  |
08:31:33 web.1  | npm ERR! Please include the following file with any support request:
08:31:33 web.1  | npm ERR!     /Users/rutger/Dropbox/projects/classmere/api/npm-debug.log
08:31:33 web.1  | exited with code 1
08:31:33 system | sending SIGTERM to all processes
```

**Comment by brandonlee503** [comment]
Hmm not sure if you wanted me to just hotfix into master and deploy so I was cautious and put it in a temp hotfix branch - https://github.com/classmere/api/tree/Issue-9-Empty-Array-Error

Just checked for the # of rows returned.  I think you might want it to be more pr0 with query.on('error' ...
stuff but this runs and catches the error too for now.

**Comment by rutgerfarry** [comment]
Closed by 9c8b65e
